app/views/account/projects/_show_tasks.html.slim

```diff
@@ -1,16 +1,13 @@
-.level id="complete_task_#{task.id}"
+.level class="task_#{task.id} #{'pending' if task.pending?}"
```

Comment:
why name of partial is "show_tasks"?
only one is rendered
rename to "_task" or something like that and it must be singular